website/content/commands/connect/redirect-traffic.mdx

```diff
@@ -36,6 +36,8 @@ Usage: `consul connect redirect-traffic [options]`
 
 #### Options for Traffic Redirection Rules
 
+- `-consul-dns-ip` - The IP address of the Consul DNS resolver. If provided, Consul DNS queries will be redirected to the provided IP address for resolving.
```

Comment:
```suggestion
- `-consul-dns-ip` - The IP address of the Consul DNS resolver. If provided, DNS queries will be redirected to the provided IP address for name resolution.
```
An iptables rule will be installed that redirects all DNS queries to the provided IP address, not just queries for the `.consul` TLD.